src/containers/PrimeStatusBanner/index.tsx

```diff
@@ -317,9 +344,9 @@ const PrimeStatusBanner: React.FC<PrimeStatusBannerProps> = props => {
   return (
     <PrimeStatusBannerUi
       xvs={xvs!}
-      claimedPrimeTokenCount={claimedPrimeTokenCount}
-      primeTokenLimit={primeTokenLimit}
-      primeClaimWaitingPeriodSeconds={primeClaimWaitingPeriodSeconds}
+      claimedPrimeTokenCount={claimedPrimeTokenCount || 0}
+      primeTokenLimit={primeTokenLimit || 0}
+      primeClaimWaitingPeriodSeconds={primeClaimWaitingPeriodSeconds || 0}
```

Comment:
We shouldn't display any UI until all the data necessary has been fetched, otherwise we're at risk of displaying incorrect temporary states. For example say every finished fetching but `primeClaimWaitingPeriodSeconds` did not, then we might show to users that they can claim their Prime token just before the UI automatically gets updated to say they need to wait 90 days before they can claim it.

To avoid the issue, I'd recommend merging all the query hooks of this PR into one (wrapping all the contract calls with `Promise.all` so they get automatically batched by multicall). Something like `getPrimeStatus`.